modules/oauth-configuring-internal-oauth.adoc

```diff
@@ -34,7 +34,7 @@ spec:
 ----
 <1> Set `accessTokenMaxAgeSeconds` to control the lifetime of access tokens.
 The default lifetime is 24 hours, or 86400 seconds. This attribute cannot
-be negative.
+be negative. If set to zero, the default lifetime will be used.
```

Comment:
will be -> is

Reply:
@slaskawi Let me know if you can make this minor change or if you want me to take over the PR to do it. Thanks!

Reply:
Fixed! Thanks for pointing out.